vars/createTestResultsMessage.groovy

```diff
@@ -1,14 +1,15 @@
 def call(Map args = [:]) {
-    def lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
+    // def lib = library(identifier: "jenkins@20211123", retriever: legacySCM(scm))
 
     String testType = args.testType
     String status = args.status
     String absoluteUrl = args.absoluteUrl
     String icon = status == 'SUCCESS' ? ':white_check_mark:' : ':warning:'
 
-    lib.jenkins.Messages.new(this).add(
-        "${STAGE_NAME}",
-        lib.jenkins.Messages.new(this).get(["${STAGE_NAME}"]) + 
-        "\n${testType}: ${icon} ${status} ${absoluteUrl}"
-    )
+    return "\n${testType}: ${icon} ${status} ${absoluteUrl}"
+    // lib.jenkins.Messages.new(this).add(
+    //     "${STAGE_NAME}",
+    //     lib.jenkins.Messages.new(this).get(["${STAGE_NAME}"]) + 
+    //     "\n${testType}: ${icon} ${status} ${absoluteUrl}"
+    // )
```

Comment:
Is this commented out by mistake?

Reply:
No. This is intentional to fix the concurrency issue https://github.com/opensearch-project/opensearch-build/pull/1807#issuecomment-1080374963 Let me delete the commented code to avoid confusion.